test/namedtuple.jl

```diff
@@ -276,3 +276,7 @@ let nt0 = NamedTuple(), nt1 = (a=33,), nt2 = (a=0, b=:v)
     @test Base.setindex(nt1, "value", :a) == (a="value",)
     @test Base.setindex(nt1, "value", :a) isa NamedTuple{(:a,),<:Tuple{AbstractString}}
 end
+
+# Pair constructor
+@test NamedTuple(:a => 1, :b => 2.0) === (a=1, b=2.0)
+@test_throws MethodError NamedTuple("a" => 1, "b" => 2.0)
```

Comment:
should add a test for `(a => 1, b => 2.0)`?

Reply:
I'm not sure what value this test adds